.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- The title should folllow Angular Commit Message Conventions (https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) -->
```

Comment:
This is news to me, though I see it's in [semantic-release readme](https://github.com/semantic-release/semantic-release#commit-message-format). 

Do we actually use the footer aspect? (I had never seen it before.) I thought deciding the version number was done via the commit type, not BREAKING CHANGE. It is nice to have a place for `fixes #...` though.

I also feel like the scope list isn't great, but I'm not sure what scopes we want to list.

Reply:
@edemaine 
> Do we actually use the footer aspect? (I had never seen it before.) I thought deciding the version number was done via the commit type, not BREAKING CHANGE.

The footer decides whether the major version (minor for `v0.x.x`) should be bumped.

> I also feel like the scope list isn't great, but I'm not sure what scopes we want to list.

I think scope usually is the name of the file that is updated. IMHO, it wouldn't matter much since our repo is small, yet.